### PR TITLE
Declare required JetStream resources in test

### PR DESCRIPTION
**Pull request by variadico**
Currently, tests that use js-op.conf use dynamic JetStream limits,
calculated from the machine where the test is run. However, this file
also has an account JWT that requires an explicit amount of resources.
This causes some tests to pass on some machine, but not others.

This change explicitly declares the max storage and memory limits
JetStream requires to match the limits specified in the account JWT, so
that the host environment doesn't influence the tests.

tl;dr; Dynamic limits correctly determines MaxStore should be 2.75GB for my Linux laptop, but the account requires 10GB. This causes the test to fail. For `/tmp/nats/jetstream`, my laptop only has 3.8GB available because tmpfs is in memory, not disk.

/cc @nats-io/core

### CODE REVIEW

**Review by derekcollison — APPROVED** [review]
LGTM - Thanks!

**Review by kozlovic — APPROVED** [review]
LGTM
